Parking/ParkingGroup/doc/spec.md

```diff
@@ -59,6 +59,24 @@ per group type.
     + Normative References: [https://schema.org/description](https://schema.org/description)
     + Optional
 
++ `status` : Status of the parking group.
+    + Attribute type: List of [Text](http://schema.org/Text)
+    + Metadata:
+        + `timestamp` : Timestamp of the last attribute update
+        + Type: [DateTime](https://schema.org/DateTime)
+    + Allowed values: The following defined by the following enumerations defined by DATEX II version 2.3 :
+        + *ParkingSiteStatusEnum*
+        + *OpeningStatusEnum*
+        + (`open`, `closed`, `closedAbnormal`,`openingTimesInForce`, `full`,
+           `fullAtEntrance`, `spacesAvailable`, `almostFull`)
+        + Or any other application-specific
+    + Optional
+
++ `floorNumber` : For parking sites with multiple floor levels, floor of this group.
```

Comment:
I am referring to floor level. I did it that way try to expliciting that it needs to be a number, because in some buildings the levels are marked with letters: A, B, C, etc.

About the value specification, it's done in the `Allowed values` field.